src/Service.Tests/SqlTests/RestApiTests/Find/MsSqlFindApiTests.cs

```diff
@@ -462,6 +462,80 @@ public class MsSqlFindApiTests : FindApiTestBase
                 $"SELECT [id], [category], [revenue], [accessible_role] FROM { _tableWithSecurityPolicy } " +
                 $"WHERE [id] = 3 " +
                 $"FOR JSON PATH, INCLUDE_NULL_VALUES, WITHOUT_ARRAY_WRAPPER"
+            },
+            {
+                "FindByIdTestWithSelectFieldsOnViewWithoutKeyFields",
+                $"SELECT [title] FROM { _simple_all_books } " +
+                $"WHERE [id] = 1 FOR JSON PATH, INCLUDE_NULL_VALUES, WITHOUT_ARRAY_WRAPPER"
```

Comment:
It would be easier to read if you bump this segment to a new line: (this comment applies to multiple tests below this comment, and makes test entries consisten with the tests already present.)
`FOR JSON PATH, INCLUDE_NULL_VALUES, WITHOUT_ARRAY_WRAPPER`